releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -14,6 +14,130 @@
 
 [Source code][source_code_web] | [Package (Maven)][package_web]
 
+## 5.9.0-beta.1 - January 11, 2024
+
+### Features Added
+
+* Added support for loading Feature Variants from Azure App Configuration.
+
+## 5.8.0 - December 13, 2023
+
+* This release is compatible with Spring Boot 3.0.0-3.0.13, 3.1.0-3.1.6, 3.2.0-3.2.0.
+* This release is compatible with Spring Cloud 2022.0.0-2022.0.4, 2023.0.0-2023.0.0.
```

Comment:
Do we need to add spring compatibility information in our release notes? I think SDK release notes should be the source of truth. We could add a link to SDK release notes if needed.